Cargo.lock

```diff
@@ -111,7 +111,7 @@ dependencies = [
 
 [[package]]
 name = "cargo-vet"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
I'm a bit surprised our `Cargo.lock` wasn't already updated. I guess we hadn't run tests since the version bump?

Reply:
yeah this through me off, i guess CI is locked so it doesn't notice